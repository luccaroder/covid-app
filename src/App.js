import React from 'react';
import logo from './logo.svg';
import './App.css';
import Monitor from './components/covid/Monitor'

const App = () => {
  return (
    <div className="App">
     <Monitor>
     </Monitor>
    </div>
  );
}

export default App;
