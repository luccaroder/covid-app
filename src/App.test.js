import React from 'react';
import App from 'App';

afterEach(cleanup)

jest.mock('components/covid/Monitor', () => 'mock-monitor');

test('renders App react component', () => {
    const { asFragment } = render(<App />)
    expect(asFragment(<App />)).toMatchSnapshot()
});
