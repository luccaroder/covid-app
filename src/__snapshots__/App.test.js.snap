// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App react component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <mock-monitor />
  </div>
</DocumentFragment>
`;
